.github/workflows/ci.yml

```diff
@@ -41,6 +41,8 @@ jobs:
     container: crystallang/crystal:${{ matrix.crystal_version }}-alpine
     steps:
       - uses: actions/checkout@v2
+      - name: Install shards
+        run: shards install
```

Comment:
It needs to go after the cache step or the cache will overwrite it and the shards will always install (worst of both worlds 😆)

Reply:
I copied it from Lucky https://github.com/luckyframework/lucky/blob/master/.github/workflows/ci.yml#L27-L34 so does that mean that's wrong too? 🤦

Reply:
Most definitely! We're real professionals haha

Reply:
Lovely! Looks like just about all of them are wrong 😂  Might as well start with this one.